views-pebble/src/main/java/io/micronaut/views/pebble/PebbleViewsRenderer.java

```diff
@@ -39,22 +39,20 @@
 @Requires(classes = PebbleEngine.class)
 public class PebbleViewsRenderer<T> implements ViewsRenderer<T> {
     
-    private final String extension;
     private final PebbleEngine engine;
 
     @Inject
-    public PebbleViewsRenderer(PebbleConfiguration configuration, PebbleEngine engine) {    
```

Comment:
This is a breaking change. You will need to keep the previous constructor remove `@Inject` from the deprecated constructor add the new constructor, annotate the new constructor with `@Inject` and annotate the old constructor with `@Deprecated`. Also, add in the old constructor javadoc instructions to migrate.

Reply:
Done except instructional javadoc